src/main/java/org/prebid/server/bidder/smaato/SmaatoBidder.java

```diff
@@ -226,21 +230,21 @@ private BidRequest modifyBidRequest(BidRequest bidRequest, String publisherId, S
         return bidRequestBuilder.imp(impSupplier.get()).build();
     }
 
-    private List<Imp> modifyImpsForAdBreak(List<Imp> imps, String adBreakId) {
+    private List<Imp> modifyImpsForAdBreak(List<Imp> imps, String adBreakId, ObjectNode impExtSkadn) {
         return IntStream.range(0, imps.size())
-                .mapToObj(idx -> modifyImpForAdBreak(imps.get(idx), idx + 1, adBreakId))
+                .mapToObj(idx -> modifyImpForAdBreak(idx, imps.get(idx), idx + 1, adBreakId, impExtSkadn))
                 .toList();
     }
 
-    private Imp modifyImpForAdBreak(Imp imp, Integer sequence, String adBreakId) {
+    private Imp modifyImpForAdBreak(Integer idx, Imp imp, Integer sequence, String adBreakId, ObjectNode impExtSkadn) {
         final Video modifiedVideo = imp.getVideo().toBuilder()
                 .sequence(sequence)
                 .ext(mapper.mapper().createObjectNode().set("context", TextNode.valueOf("adpod")))
                 .build();
         return imp.toBuilder()
                 .tagid(adBreakId)
                 .video(modifiedVideo)
-                .ext(null)
+                .ext(idx == 0 ? impExtSkadn : null)
```

Comment:
No need to pass additional parameter and check it here, can be done one layer before.